rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/tsdb/130_position_fields.yml

```diff
@@ -0,0 +1,316 @@
+---
+setup:
+  - skip:
+      version: " - 8.7.99"
+      reason: position metric introduced in 8.8.0
+  - do:
+      indices.create:
+        index: locations
+        body:
+          settings:
+            index:
+              mode: time_series
+              routing_path: [ city, name ]
+              time_series:
+                start_time: 2023-01-01T00:00:00Z
+                end_time: 2024-01-01T00:00:00Z
+          mappings:
+            properties:
+              "@timestamp":
+                type: date
+              city:
+                type: keyword
+                time_series_dimension: true
+              name:
+                type: keyword
+                time_series_dimension: true
+              location:
+                type: geo_point
+                time_series_metric: position
+
+  - do:
+      bulk:
+        index: locations
+        refresh: true
+        body: |
+          {"index":{}}
+          {"@timestamp": "2023-01-01T12:00:00Z", "location": "POINT(4.912350 52.374081)", "city": "Amsterdam", "name": "NEMO Science Museum"}
+          {"index":{}}
+          {"@timestamp": "2023-01-02T12:00:00Z", "location": "POINT(4.901618 52.369219)", "city": "Amsterdam", "name": "Museum Het Rembrandthuis"}
+          {"index":{}}
+          {"@timestamp": "2023-01-03T12:00:00Z", "location": "POINT(4.914722 52.371667)", "city": "Amsterdam", "name": "Nederlands Scheepvaartmuseum"}
+          {"index":{}}
+          {"@timestamp": "2023-01-04T12:00:00Z", "location": "POINT(4.405200 51.222900)", "city": "Antwerp", "name": "Letterenhuis"}
+          {"index":{}}
+          {"@timestamp": "2023-01-05T12:00:00Z", "location": "POINT(2.336389 48.861111)", "city": "Paris", "name": "Musée du Louvre"}
+          {"index":{}}
+          {"@timestamp": "2023-01-06T12:00:00Z", "location": "POINT(2.327000 48.860000)", "city": "Paris", "name": "Musée dOrsay"}
+  - do:
+      indices.refresh: { }
+
+---
+multi-valued fields unsupported:
+  - skip:
+      version: " - 8.7.99"
+      reason: position metric introduced in 8.8.0
+  - do:
+      bulk:
+        index: locations
+        refresh: true
+        body: |
+          {"index" : {}}
+          {"@timestamp": "2023-01-01T12:00:00Z", "city": "Rock", "name": "On", "location": {"lat": 13.0, "lon" : 34.0} }
+          {"index" : {}}
+          {"@timestamp": "2023-01-01T12:00:00Z", "city": "Hey", "name": "There", "location": [{"lat": 13.0, "lon" : 34.0}] }
+          {"index" : {}}
+          {"@timestamp": "2023-01-01T12:00:00Z", "city": "Boo", "name": "Hoo", "location": [{"lat": 13.0, "lon" : 34.0}, {"lat": 14.0, "lon" : 35.0}] }
+  - match: { errors: true }
+  - match: { items.0.index.result: "created" }
+  - match: { items.1.index.result: "created" }
+  - match: { items.2.index.error.reason: "failed to parse" }
+  - match: { items.2.index.error.caused_by.reason: "field type for [location] does not accept more than single value" }
```

Comment:
Do we have information here to add that the field is a position metric?

Reply:
I'll take a look. I originally wrote the error to mimic similar errors on other metrics, but see no reason why I should not add this fact to the message.

Reply:
Hmm... trickier than expected. The place where I throw this exception does not know about position metrics, only that multi-values are disallowed. I would need to pass more information down the stack to achieve this.

Reply:
OK. I did not make a change here, since I thought passing the MetricType down to the parser seemed a bit odd. I don't have a strong opinion on this, so if you still want me to change the error message, I can still make that change.